tests/v2/validation/provisioning/rke2/etcd_backup_restore.go

```diff
@@ -231,3 +231,21 @@ func watchAndWaitForPods(client *rancher.Client, clusterID string) error {
 	})
 	return err
 }
+
+func upgradeClusterK8sVersionWithUpgradeStrategy(client *rancher.Client, clustername string, k8sUpgradedVersion string, namespaceName string) error {
+	clusterObj, existingSteveAPIObj, err := getProvisioningClusterByName(client, clustername, namespaceName)
+	if err != nil {
+		return err
+	}
+
+	clusterObj.Spec.RKEConfig.UpgradeStrategy.ControlPlaneConcurrency = "10%"
+	clusterObj.Spec.RKEConfig.UpgradeStrategy.WorkerConcurrency = "20%"
```

Comment:
doesn't both of these settings values default to 10% ? `ControlPlaneConcurrency` is not going to change after upgrade is it ?

Reply:
right, i've updated the ControlPlaneConcurrency value to 15% now.